### ISSUE TITLE
Big Sur support incomplete - BaseSystem.dmg not found

### ISSUE THREAD

**Issue by Erisa** [foxlet/macOS-Simple-KVM#250]
Hi I'm sorry if you're already working on this but I wanted to make you (And others) aware that the current Big Sur support added in https://github.com/foxlet/macOS-Simple-KVM/commit/e6c0b56fb70a264b6143e273443039e99f4af286 incorrectly attempts to convert `BaseSystem.dmg` without first extracting one from the downloaded `InstallAssistant.pkg`.

**Comment by foxlet** [comment]
Yeah, full support isn't available yet. It's mostly there for researchers/archivists trying to backup the installer data.

**Comment by Erisa** [comment]
> The easiest way for users to upgrade currently is to backup their boot disk image, then upgrade from Catalina.
Yeah that makes sense, I'm trying to install fresh to Big Sur so having to poke in the files a little. Expecting it to not work properly, just curious what I can get to work.
Thanks :)

**Comment by foxlet** [comment]
It should work fine in theory, all the hardware is natively handled, not through kexts.

**Comment by foxlet** [comment]
... and the theory is correct!

![Screenshot_20200623_040615](https://user-images.githubusercontent.com/1627533/85377968-e20b2080-b507-11ea-91a2-3269a9f8bbcf.png)
![Screenshot_20200623_040941](https://user-images.githubusercontent.com/1627533/85377952-dddf0300-b507-11ea-89fa-fb56cc31ef53.png)

**Comment by Erisa** [comment]
I also had tried it last night with virt-manager and the same template as Catalina and got this:
![260-Xming_1282x818_Mediumforestgreen](https://user-images.githubusercontent.com/14004943/85432875-7fec0300-b57b-11ea-80e1-d59bea555935.png)
(Virt Manager is X forwarded to my main vfio Windows VM for easy management pre-install)

After that attempt I ended up installing Catalina, havent tried upgrading yet.

**Comment by foxlet** [comment]
> After that attempt I ended up installing Catalina, havent tried upgrading yet.

macOS-Simple-KVM will need to be modified, the version in `master` is not compatible yet.

**Comment by mailinglists35** [comment]
> 
> 
> I also had tried it last night with virt-manager and the same template as Catalina and got this:
> ![260-Xming_1282x818_Mediumforestgreen](https://user-images.githubusercontent.com/14004943/85432875-7fec0300-b57b-11ea-80e1-d59bea555935.png)
> (Virt Manager is X forwarded to my main vfio Windows VM for easy management pre-install)
> 
> After that attempt I ended up installing Catalina, havent tried upgrading yet.

I get the same screen after the first reboot (setup is a two part process: first it copies setup files, then the boot. It completes copying the files, but on reboot I get the same screen)

**Comment by mailinglists35** [comment]
on other hackintosh forums people complain as well, I think clover needs to be updated.

**Comment by mailinglists35** [comment]
this appears to be the issue https://github.com/CloverHackyColor/CloverBootloader/issues/190

**Comment by mailinglists35** [comment]
> 
> 
> ... and the theory is correct!
> 
> ![Screenshot_20200623_040615](https://user-images.githubusercontent.com/1627533/85377968-e20b2080-b507-11ea-91a2-3269a9f8bbcf.png)
> ![Screenshot_20200623_040941](https://user-images.githubusercontent.com/1627533/85377952-dddf0300-b507-11ea-89fa-fb56cc31ef53.png)

how did you get past the clover error? is that a vm?
